trymito.io/pages/spreadsheet-automation.tsx

```diff
@@ -97,6 +97,28 @@ const SpreadsheetAutomation: NextPage = () => {
                 <Image src={'/explore_data_visually.png'} alt='Explore data with Mito' width={500} height={250} layout='responsive'/>
               </div>
             </div>
+            <div className={pageStyles.subsection}>
+              <div className={classNames(textImageSplitStyles.functionality_media, 'display-desktop-only-inline-block')}>
+                <Image src={'/export_to_excel.png'} alt='Mito generated code' width={500} height={250} layout='responsive'/>
+              </div>
+              <div className={textImageSplitStyles.functionality_text}>
+                <h1>
+                  Compatible with <span className='text-highlight'>Excel</span>
```

Comment:
I think using "Compatible" sells ourself a bit short. Everything is compatible with Excel in some form! 

Our unique value is that we can generate Excel files. So maybe we could go with something like:

1. Create presentation-ready Excel files 
2. Generate formatted Excel files